openapi-spec/swagger.yaml

```diff
@@ -1188,6 +1193,131 @@ paths:
           description: The resource does not exist.
         '500':
           description: The fileshare can not be deleted. please check status
+
+  '/v1/{tenantId}/tiers':
```

Comment:
is this final to call tiers? Labels for these APIs are "Storage Service Plan"

Reply:
Yeah...for now, it's little bit confusing...bcz all API was built, then we changed the name to SSP. But later we will update this